theme.html

```diff
@@ -0,0 +1,95 @@
+        <img src="./src/img/docs-logo.png"/>
+    </a>
+
+    <div class="sidebar">
+      <li class="list-head">Getting Started</li> 
+      <a href="./docs.html"><li>Introduction</li></a>
+       <a href="./installation.html"> <li>Installation</li> </a>
+       <a href="./typo.html"> <li>Typography</li> </a>
+       <a href="./theme.html"> <li class="active">Theme</li> </a>
+      
+      <li class="list-head">UI Components</li>      
+       <a href="./src/components/alert/alert.html"> <li>Alert</li> </a>
+       <a href="./src/components/avatar/avatar.html"> <li>Avatar</li> </a>
+       <a href="./src/components/badge/badge.html"> <li>Badge</li> </a>
+       <a href="./src/components/button/button.html"> <li>Button</li> </a>
+       <a href="./src/components/card/card.html"> <li>Card</li> </a>
+       <a href="./src/components/grid/grid.html"> <li>Grid</li> </a>
+       <a href="./src/components/image/image.html"> <li>Image</li> </a>
+       <a href="./src/components/input/input.html"> <li>Input</li> </a>
+       <a href="./src/components/lists/lists.html"> <li>Lists</li> </a>
+       <a href="./src/components/modal/modal.html"> <li>Modal</li> </a>
+       <a href="./src/components/navigation/navigation.html"> <li>Navigation</li> </a>
+       <a href="./src/components/rating/rating.html"> <li>Rating</li> </a>
+       <a href="./src/components/toast/toast.html"> <li>Toast</li> </a>
+      </div>
+</aside>
+<main class="doc-main"> 
+    <nav>
+        <ul class="nav-group start">
+          <li class="nav-active"> <a href="./docs.html">Getting Started</a></li>
+            <li><a href="./src/components/alert/alert.html">UI Components</a></li>
+        </ul>
+        <ul class="nav-group end">
+        <li><a href="https://github.com/shangkaul/krozaro_component_lib"><i class="icon icon-md fa fa-github"></i></a></li>
+        <li></li>
+    </ul>
+    
+    </nav>
+    <div class="doc-content">
+        <section class="component">
+            <h1>Theme</h1>
+          <p class="md"> The Krozaro library is built to be a blank slate that can be easily customized to fit a brand. We have incorporated some default styles as CSS variables.
+            These variables can be easily customized to modify the look and feel of your application.
+          </p>  
+          </section>
+
+          <section class="component">
+            <h2>Default Color Theme</h2>
+            <section class="demo">
+              <div class="color-block" style="background-color: var(--color-primary);">
+             Primary Color
+              </div>
+              <div class="color-block" style="background-color: var(--gray-text);">
+                Gray Text
+                 </div>
+                 <div class="color-block" style="background-color: var(--gray-1);color:var(--gray-text);">
+                  Gray-1 (backgrounds)
+                   </div>
+                   <div class="color-block" style="background-color: var(--color-success);">
```

Comment:
Rather than Gray Text or Gray-1, I think color theme names should be like : Primary, Secondary, Accent, Background, Success, Error and Warning.